buildenv/jenkins/testJobTemplate

```diff
@@ -446,6 +447,7 @@ ARCH_OS_LIST.each { ARCH_OS ->
 							booleanParam('SETUP_JCK_RUN', SETUP_JCK_RUN.toBoolean(), "setup jdk during release for jck interactive run")
 							stringParam('JENKINS_KEY', JENKINS_KEY, "Optional. Only use this for Multi-node Compiler TCK tests")
 							stringParam('RELATED_NODES', RELATED_NODES, "Optional. Only use this for reserving secondary nodes other than current")
+							stringParam('ADDITIONAL_ARTIFACTS_REQUIRED', ADDITIONAL_ARTIFACTS_REQUIRED, "Optional. Only use this to specify the type of additional artifacts")
```

Comment:
On second thought, should we add `ADDITIONAL_ARTIFACTS_REQUIRED` after `CUSTOMIZED_SDK_URL_CREDENTIAL_ID` https://github.com/adoptium/aqa-tests/blob/587140d446f3bdb6847e38bd2eb1fe6c647ded87/buildenv/jenkins/testJobTemplate#L320. Ideally, it will be used with `CUSTOMIZED_SDK_URL`.